optimum/intel/openvino/configuration.py

```diff
@@ -54,7 +54,7 @@ def __init__(
         self.compression = compression or DEFAULT_QUANTIZATION_CONFIG
         self.input_info = input_info
         self.save_onnx_model = save_onnx_model
-        self._enable_standard_onnx_export_option()
+        self._set_standard_onnx_export_option()
```

Comment:
This method is currently in use by users. Is renaming necessary?

Reply:
I felt that it is a misnomer. The method sets "export_to_onnx_standard_ops" in the compression config and the value depends on instance initialization argument "save_onnx_model" which can be true or false, it is only enabled when it is true.

Reply:
Both names look meaningful. Moreover, this is not a public API, I would recommend not changing the initial name since there are concerns.

Reply:
Reverted